deployer/loader.go

```diff
@@ -18,9 +18,8 @@ import (
 
 // StateLoader struct
 type StateLoader struct {
-	// TODO: support more than one deployment per node
-	currentNodeDeployment map[uint32]uint64
-	currentNodeNetwork    map[uint32]uint64
+	currentNodeDeployment map[uint32][]uint64
```

Comment:
that's a nested map but honestly from the type I don't know the intention, can be simplified using alias declarations